.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: go
+
+sudo: true
+
+go:
+- 1.4
+- tip
+
+before_script:
+- go build keywhizfs/main.go
+- sudo apt-get install fuse-utils
```

Comment:
Is installing fuse-utils (and `sudo: true`) necessary since fs_test.go is excluded?